src/html.rs

```diff
@@ -81,7 +81,7 @@ pub fn has_attr(el: &NodeRef, attr_name: &str) -> bool {
     attributes.get(LocalName::from(attr_name)).is_some()
 }
 
-pub fn get_attr(el: &NodeRef, attr_name: &str) -> Option<String> {
+pub fn get_attr_val(el: &NodeRef, attr_name: &str) -> Option<String> {
```

Comment:
Same, I think `get_attribute` would be the most consistent (because the equivalent DOM API is `getAttribute(name)`).